packages/amplify-e2e-tests/src/__tests__/rds-mysql-auth-apikey-lambda.test.ts

```diff
@@ -27,7 +27,7 @@ describe('RDS Relational Directives', () => {
   // Generate settings for RDS instance
   const username = db_user;
   const password = db_password;
-  let region = 'ap-northeast-2';
+  let region = 'us-east-1';
```

Comment:
We're talking about allow listing our E2E account to test this in ICN first. Can keep this the same.